### ISSUE TITLE
Running JBang as a Scratch file will not set the Script file attribute properly

### ISSUE THREAD

**Issue by nandorholozsnyak** [jbangdev/jbang-idea#68]
**Bug description:**
I really like the IDEA's Scratch feature where devs can make different types of files and if it is a runnable thing, like a Java class than it could be ran as well.
When the JBang's shebang directive is configured then the IDEA recognizes it as a JBang script and we can see the shiny JBang icon to run the script, but it could not be run because the newly created **Run Configuration's** `Script file` attribute is not set properly.

![image](https://user-images.githubusercontent.com/11406183/188558785-a1ca1376-075d-4190-9b61-1e427fe75eb8.png)

Hitting on the JBang icon produces the following output in a console window:
```
/home/xxx/.jbang/bin/jbang run Idea2022.2/scratches/JBang/scratch_15.java
[jbang] [ERROR] Script or alias could not be found or read: 'Idea2022.2/scratches/JBang/scratch_15.java'
[jbang] Run with --verbose for more details
```

Opening the Run configuration is not setting the full path to the scratch file:
![image](https://user-images.githubusercontent.com/11406183/188558948-db789b73-7980-4764-ae1d-4cb56b03bad5.png)

**Expected behaviour:**
Plugin should set the `Script file` attribute to the absolute path of the scratch file, in my case:
`/home/xxx/.config/JetBrains/IntelliJIdea2022.2/scratches/JBang/scratch_15.java`

**Comment by linux-china** [comment]
It's a bug caused by absolute path, and I fixed it. 

![image](https://user-images.githubusercontent.com/46711/188784248-20133108-5692-4985-885c-3a178527cf35.png)

**Comment by nandorholozsnyak** [comment]
Thank you

**Comment by nandorholozsnyak** [comment]
Fixed in #69, thank you :)
